target/linux/bmips/dts/bcm6328-comtrend-ar-5387un.dts

```diff
@@ -0,0 +1,212 @@
+/dts-v1/;
+
+#include "bcm6328.dtsi"
+
+/ {
+	compatible = "comtrend,ar-5387un", "brcm,bcm6328";
+	model = "Comtrend AR-5387un";
+
+	chosen {
+		bootargs = "console=ttyS0,115200 earlycon";
+	};
+
+	aliases {
+		led-boot = &power_green;
+		led-failsafe = &power_red;
+		led-running = &power_green;
+		led-dsl = &dsl_green;
+		led-internet = &inet_green;
+	};
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x00000000 0x04000000>;
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+
+		reset {
+			label = "reset";
+			gpios = <&pinctrl 23 1>;
+			linux,code = <KEY_RESTART>;
+			debounce-interval = <60>;
+		};
+	};
+
+	bcm43225-sprom {
+		compatible = "brcm,bcm43225-sprom";
+
+		pci-bus = <1>;
+		pci-dev = <0>;
+
+		mtd-mac-address = <&cfe 0x6a0>;
+		mtd-mac-address-increment = <1>;
+
+		brcm,sprom-fixups = <2 0x05bb>,
+				    <65 0x1204>,
+				    <78 0x0202>,
+				    <79 0x0202>,
+				    <80 0xff02>,
+				    <87 0x0315>,
+				    <88 0x0315>,
+				    <96 0x2048>,
+				    <97 0xff11>,
+				    <98 0x1567>,
+				    <99 0xfb24>,
+				    <100 0x3e3c>,
+				    <101 0x4038>,
+				    <102 0xfe7f>,
+				    <103 0x1279>,
+				    <112 0x2048>,
+				    <113 0xff03>,
+				    <114 0x154c>,
+				    <115 0xfb27>,
+				    <116 0x3e3c>,
+				    <117 0x4038>,
+				    <118 0xfe87>,
+				    <119 0x1233>,
+				    <203 0x2226>;
+	};
+};
+
+&ehci {
+	status = "ok";
+};
+
+&enet {
+	status = "okay";
+
+	mtd-mac-address = <&cfe 0x6a0>;
+
+	port@0 {
+		compatible = "brcm,enetsw-port";
+		reg = <0>;
+		label = "fe1";
+		brcm,phy-id = <1>;
+	};
+
+	port@1 {
+		compatible = "brcm,enetsw-port";
+		reg = <1>;
+		label = "fe2";
+		brcm,phy-id = <2>;
+	};
+
+	port@2 {
+		compatible = "brcm,enetsw-port";
+		reg = <2>;
+		label = "fe3";
+		brcm,phy-id = <3>;
+	};
+
+	port@3 {
+		compatible = "brcm,enetsw-port";
+		reg = <3>;
+		label = "fe4";
+		brcm,phy-id = <4>;
+	};
+};
+
+&hsspi {
+	status = "okay";
+
+	m25p80@0 {
+		compatible = "jedec,spi-nor";
+		spi-max-frequency = <16666667>;
+		spi-tx-bus-width = <2>;
+		spi-rx-bus-width = <2>;
+		reg = <0>;
+
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			cfe: cfe@0 {
+				reg = <0x000000 0x010000>;
+				label = "cfe";
+				read-only;
+			};
+
+			firmware@10000 {
+				reg = <0x010000 0xfe0000>;
+				label = "firmware";
+				compatible = "brcm,bcm963xx-imagetag";
+			};
+
+			nvram@ff0000 {
+				reg = <0xff0000 0x010000>;
+				label = "nvram";
+			};
+		};
+	};
+};
+
+&leds {
+	status = "ok";
+
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_leds
+		     &pinctrl_ephy0_act_led &pinctrl_ephy1_act_led
+		     &pinctrl_ephy2_act_led &pinctrl_ephy3_act_led>;
+
+	inet_red: led@1 {
+		reg = <1>;
+		label = "AR-5387un:red:inet";
```

Comment:
Don't know whether the label is constrained by something, but if you start from scratch it would be convenient to use the model part of the device name here: "ar-5387un:red:internet"
This has the advantage that it can be derived from device name if necessary.